src/parse-apigw-http-event.js

```diff
@@ -14,7 +17,7 @@ module.exports = function parseApiGwHttpEvent(event){
   }
 
   return {
-    sourceType: 'api',
+    sourceType: ApiGateway,
```

Comment:
This feels like an odd way to use a constant... Instead import event and reference Event.ApiGateway